list/dup.php

```diff
@@ -0,0 +1,26 @@
+<?php
+	require_once "../common.php";
+	require_login();
+
+    $new_name = $_POST['name_change_field'];
+    $original_id = $_POST['og_plan_id'];
+
+    $og_plan_check = $db->query("SELECT plan_title
+                                FROM plan
+                                WHERE user_id = ? AND plan_id = ?", [$_SESSION["user_id"],$original_id]);
+    if (count($og_plan_check) == 1)
+    {
+        $db->query("INSERT INTO 
+                    plan (plan_title, user_id, degree_id, plan_status, json)
+                        SELECT ?, user_id, degree_id, plan_status, json
```

Comment:
Interesting, I don't think I've ever put a ? in the select clause before, but it looks like a good solution for this use-case!